src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const res = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
Тут можно использовать сокращенную запись (если свойство === название переменной, name === name, wheels === wheels)
```suggestion
    name,
    wheels,
    version,
```